server/server.js

```diff
@@ -63,6 +65,9 @@ app.use(
 app.use(busboy());
 app.use(bodyParser.json());
 app.use(express.static(STATIC_PATH));
+app.use('/l20n', express.static(L20N));
+app.use('/download/*/l20n', express.static(L20N));
+app.use('/download/*/locales', express.static(LOCALES));
```

Comment:
The express app doesn't recognize the static route if you don't explicitly add routes with respect to /download.